### PR TITLE
Swish, Exponential, Hard Sigmoid Activation Functions

### PR DESCRIPTION
**Pull request by SheepSeb**
## Pull Request Overview

Improvment in adding more activation functions from tensorflow

### Is this a breaking change?
No 
### Are all new or changed code paths covered by unit testing?
Yes
### A complete listing of issues addressed or closed with this change.
None
### A complete listing of any enhancements provided by this change.
Implementation of the follwing activation functions:
- Swish
- Exp
- Hard Sigmoid
### Any usage details developers may need to make use of this new functionality.
No additional usage details
### Does additional documentation need to be developed beyond what is listed in your Pull Request?
No additional documentation needed
### Any other salient points of interest.
No

### CODE REVIEW

**Comment by makr11st** [comment]
@SheepSeb Thank you for your contribution, we'll review it as soon as possible and merge soon.

File: src/activations.rs [hunk 1]
```diff
@@ -314,6 +395,27 @@ mod tests {
...
+        assert_eq!(expected,result);
+    }
+
+    #[test]
+    fn test_switsh_2d(){
+        let data:Array2<f32> = Array2::from_shape_vec((2,2), vec![-2.0, 2.0, 4.0, -7.0]).unwrap();
+        
+        let result:Array2<f32> = Activation::Swish.activation(&data);
+        let expected:Array2<f32> = array![[-0.23840584,  1.761594],
+                                        [3.92805516, -0.006377358]];
```
Comment (makr11st) [inline src/activations.rs]: If you please apply `rustfmt` so we have all the formatting consistent.
```suggestion
        let data: Array2<f32> = Array2::from_shape_vec((2,2), vec![-2.0, 2.0, 4.0, -7.0]).unwrap();
        
        let result: Array2<f32> = Activation::Swish.activation(&data);
        let expected: Array2<f32> = array![[-0.23840584,  1.761594],
                                        [3.92805516, -0.006377358]];
```

**Review by makr11st — COMMENTED** [review]
Just a minor formatting issue, please run `rustfmt`. 
Otherwise LGTM and we can merge as soon as this minor formatting is ammended.

File: src/activations.rs (hunk 1 above)
Comment (SheepSeb) [reply src/activations.rs]: I've formatted and pushed the changes

**Review by makr11st — APPROVED** [review]

**Pull request merged**